src/Aspire.Hosting/Orchestrator/ParameterProcessor.cs

```diff
@@ -14,16 +15,24 @@ namespace Aspire.Hosting.Orchestrator;
 /// <summary>
 /// Handles processing of parameter resources during application orchestration.
 /// </summary>
-internal sealed class ParameterProcessor(
+[Experimental("ASPIREINTERACTION001", UrlFormat = "https://aka.ms/aspire/diagnostics/{0}")]
+public sealed class ParameterProcessor(
```

Comment:
I think this is the first public type in the `Aspire.Hosting.Orchestrator` namespace. I'm wondering if we should make an interface in a separate namespace that is already exposed.

Reply:
Good callout -- I think we should move this to the `Aspire.Hosting` namespace to match what we do for `InteractionService`.